### PR TITLE
Add test for label resolver with empty description

### PR DESCRIPTION
**Pull request by eseliger**
I figured we should have a test for that as well, to prevent a regression.

### CODE REVIEW

File: enterprise/internal/campaigns/resolvers/changeset_test.go [hunk 1]
```diff
@@ -105,12 +107,18 @@ func TestChangesetResolver(t *testing.T) {
...
 						Name:        "cool-label",
 						Color:       "blue",
-						Description: "the best label in town",
+						Description: labelEventDescriptionText,
 					},
 				}},
 			},
+			Labels: struct{ Nodes []github.Label }{
+				Nodes: []github.Label{
+					{ID: "label-no-description", Name: "Bug", Color: "121212"},
```
Comment (mrnugget) [inline enterprise/internal/campaigns/resolvers/changeset_test.go:119]:
```suggestion
					{ID: "label-no-description", Name: "no-description", Color: "121212"},
```

**Comment by eseliger** [comment]
Yes that was my intention

File: enterprise/internal/campaigns/resolvers/changeset_test.go [hunk 2]
```diff
@@ -162,7 +170,8 @@ func TestChangesetResolver(t *testing.T) {
 					TotalCount: 2,
 				},
 				Labels: []apitest.Label{
-					{Text: "cool-label", Color: "blue", Description: "the best label in town"},
+					{Text: "Bug", Color: "121212", Description: nil},
```
Comment (eseliger) [inline enterprise/internal/campaigns/resolvers/changeset_test.go:173]:
```suggestion
					{Text: "no-description", Color: "121212", Description: nil},
```

File: enterprise/internal/campaigns/resolvers/changeset_test.go (hunk 1 above)
Comment (eseliger) [reply enterprise/internal/campaigns/resolvers/changeset_test.go:119]: It's a very common label on github 😁

**Pull request merged**